finary_uapi/currencies.py

```diff
@@ -30,3 +30,12 @@ def get_cryptocurrency_by_code(session: requests.Session, code: str):
                 return currency
         return currency
     return {}
+
+def get_fiatcurrency_by_code(session: requests.Session, code: str):
+    currencies = get_currencies(session, "fiat", code)
+    if len(currencies["result"]) > 0:
+        currency = currencies["result"][0]
+        for currency in currencies["result"]:
+            if currency["code"] == code:
+                return currency
+        return currency
```

Comment:
What's your operating system and IDE ? you need to configure it to add a new line at end of file.

Reply:
VS Codeserver in a *nix docker :p I'll check if it can be added automatically, but I think it's me deleting it each time 8-p